src/blockly/generators/propc.js

```diff
@@ -711,42 +639,12 @@ Blockly.FieldVariable.dropdownCreate = function() {
 };
 
 
-// NOTE: Replaces core function!
-Blockly.BlockSvg.prototype.setCollapsed = function (b) {
```

Comment:
I believe there is one remaining reference to the setCollapsed() method in procedures.js, line 498.  Can you verify that please?

Reply:
It should remain.  Much of procedures and variables is written and copied from the blockly examples/core directly and not written by us.
That being said, since we updated the core - we probably need to check and see if the newest published versions of these block are improved from what we are currently using.

Reply:
Can you check that out? I don't think I have a handle on what is happening in this code to make a definitive call.

Reply:
Yes - there are changes (just diff'd the relevant parts of procedures.js to check against blockly release 2.20190722.0).  We should have replaced/updated block defs for procedures, controls, and variables (blocks where we simply use the code provided by google) when we did a core update.  We're okay leaving things alone right now and waiting until our next core update to include these changes, or we can update those blocks to match the core version we are currently using.  They aren't broken (we'd know - they are the most commonly used blocks), so it's a matter of preference.